h/dataservice.h

```diff
@@ -64,10 +64,12 @@ typedef struct DataService_tag {
   ExternalAPI *externalAPI;  /* a function call to some lower level module to get data */
   void        *extension;    /* another slot to stash things */
   JsonObject *serviceDefinition;
+  uint64 loggingIdentifier;
```

Comment:
This breaks all plugins that haven't been rebuilt after this commit - they will assume that the ```plugin``` field is at the old offset, whereas it's moved.